_posts/2021-01-11-GPS-serverless.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: 7 Steps to Create a Serverless Tracking Solution
+date: 2021-01-11
+tags: cloudformation serverless appsync dynamodb
+author: Shermayne Lee
+image:
+---
+
+While being a dog lover, I love spending most of my free time outdoors with my dog. However, I always have difficulty calling him back when he runs outside, and I hate when he is out of my sight. Therefore to solve a few challenges I faced when I am with him, I came out with a serverless GPS monitoring solution to track his movement.
+
+## Solution ?
+
+<center><img src="/img/blog/serverless-GPS/map.png" /></center><br/>
+
+<center><img src="/img/blog/serverless-GPS/archi.jpeg" /></center><br/>
+
+## First Step: Getting started with the physical devices
+
+In my opinion, this could be the most challenging part because there are so many devices available in the market, and I have no idea which is the right one for my solution.
+
+My rule of thumb is always getting something more comfortable to start. For instance, get a physical device that is fully assembled instead of getting every separate part and try to figure out how to build it.
```

Comment:
> My rule of thumb is always getting something more comfortable to start. For instance, get a physical device that is fully assembled instead of getting every separate part and try to figure out how to build it.

perhaps have 

My rule of thumb for starting something new is to choose the task or approach with the lowest barrier to entry. For instance, choosing a physical device that is fully assembled instead of getting every separate part and trying to figure out how to build it.